spec/features/sort_peeps_spec.rb

```diff
@@ -0,0 +1,11 @@
+feature 'Sorting peeps by date' do
+  scenario "sort peeps" do    
+    visit("/home")
+    fill_in('message', with: "I'm a new peep 1")
+     fill_in('message', with: "I'm a new peep 2")
+    fill_in('message', with: "I'm a new peep 3")
+    click_button('Post Peeps')
+    click_button('New peeps on top')
+    expect(Message.all.first.message).to eq "I'm a new peep 3"
+  end
```

Comment:
Your test is giving you a false positive, because you only ever create the last message. It should be something like this:

```ruby
  scenario "sort peeps" do    
    visit("/home")
    fill_in('message', with: "I'm a new peep 1")
    click_button('Post Peeps')

    fill_in('message', with: "I'm a new peep 2")
    click_button('Post Peeps')

    fill_in('message', with: "I'm a new peep 3")
    click_button('Post Peeps')

    click_button('New peeps on top')
    expect(Message.all.first.message).to eq "I'm a new peep 3"
  end
```